.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ on: [pull_request, push]
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Now that `ubuntu-latest` installs 22.04 Python 3.6 is no longer available in that runner, so for this build job to run I had to set up Ubuntu to 20.04.

Alternatively a newer Python version could be used.

Reply:
My vote would be to use the newer Python and stick with the latest Ubuntu, like how the CircuitPython libraries do.